scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -325,8 +325,8 @@
                                                          (conj (:task aggregation) task)
                                                          (:task aggregation))
                                                  :gpus (+ (:gpus aggregation) (or gpus 0.0))
-                                                 :mem (+ (:mem aggregation) mem)
-                                                 :cpus (+ (:cpus aggregation) cpus)})
+                                                 :mem (+ (:mem aggregation) (or mem 0))
+                                                 :cpus (+ (:cpus aggregation) (or cpus 0))})
```

Comment:
@wyegelwel It appears that mem and cpus are nil whenever task is nil. You can see that in this function, there is some code to handle the case where task is nil, but mem and cpus are assumed not to be nil.